test/e2e/gatekeeper_controller.go

```diff
@@ -46,9 +46,9 @@ import (
 
 const (
 	// The length of time between polls.
-	pollInterval = 50 * time.Millisecond
+	pollInterval = time.Second
```

Comment:
This is OK for now, but I'd prefer to leave a low default for local testing and allow it to be configurable via command line so we can set a higher value when running in CI.

Reply:
Done